doc/Getting-started.md

```diff
@@ -9,6 +9,8 @@ Let's use the REST API offered by [JSONPlaceholder](http://jsonplaceholder.typic
 
```

Comment:
I think you should revert the changes on this one, to let newcomers begin with the simplest version (`<script>` tag)